pipelines/constants.py

```
@@ -25,8 +25,8 @@ class constants(Enum):  # pylint: disable=c0103
     GCS_FLOWS_BUCKET = "datario-public"
     # PROJECT_NAME = {"dev": "rj-smtr-dev", "prod": "rj-smtr"}
     # DEFAULT_BUCKET_NAME = {"dev": "br-rj-smtr-dev", "prod": "br-rj-smtr"}
-    PROJECT_NAME = {"dev": "rj-smtr-dev", "prod": "rj-smtr-staging"}
-    DEFAULT_BUCKET_NAME = {"dev": "rj-smtr-dev", "prod": "rj-smtr-staging"}
+    PROJECT_NAME = {"dev": "rj-smtr-dev", "prod": "rj-smtr-dev"}
```

Ruff (0.13.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)